platforms/unix/src/atspi/object_address.rs

```diff
@@ -3,39 +3,39 @@
 // the LICENSE-APACHE file) or the MIT license (found in
 // the LICENSE-MIT file), at your option.
 
-use atspi::Accessible;
+use atspi::ObjectRef;
 use serde::{Deserialize, Serialize};
 use zbus::{
-    names::UniqueName,
+    names::{OwnedUniqueName, UniqueName},
     zvariant::{ObjectPath, OwnedObjectPath, OwnedValue, Type, Value},
 };
 
 const NULL_PATH: &str = "/org/a11y/atspi/null";
 
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, OwnedValue, Type, Value)]
 pub(crate) struct OwnedObjectAddress {
-    bus_name: String,
+    bus_name: OwnedUniqueName,
     path: OwnedObjectPath,
 }
 
 impl OwnedObjectAddress {
     pub(crate) fn new(bus_name: &UniqueName, path: OwnedObjectPath) -> Self {
         Self {
-            bus_name: bus_name.to_string(),
+            bus_name: bus_name.to_owned().into(),
             path,
         }
     }
 
     pub(crate) fn null() -> Self {
         Self {
-            bus_name: String::new(),
+            bus_name: UniqueName::from_static_str("").unwrap().into(),
```

Comment:
This will fail as D-Bus unique names can't be empty strings. Unless there is a reason to these changes that I don't get, please revert changes to this file.

Reply:
ok, I was irritated that the `atspi` crate uses completely different values for the "null" object address, I can use their values instead.

Reply:
specifically: https://docs.rs/atspi-common/0.6.0/src/atspi_common/object_ref.rs.html#23-32